content/en/docs/concepts/configuration/secret.md

```diff
@@ -857,9 +857,9 @@ type: kubernetes.io/tls
 data:
```

Comment:
This could be `stringData`

Reply:
Please confirm if I should still proceed with this change.

Reply:
Please do work on the pull request here. If you change to `stringData`, then AIUI the values need to _not_ change.

Reply:
I've just read the note in the screenshot below and gotten a bit confused. I've reverted the commit adding the headers in the example.

![image](https://github.com/kubernetes/website/assets/41787587/8cb99011-44cf-4082-8e71-ac71d2a52ed3)